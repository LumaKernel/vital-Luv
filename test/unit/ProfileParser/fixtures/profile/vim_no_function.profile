SCRIPT  /home/luma/vim-make-plugin/vital-Luv/dev/ProfileParser/test_no_function.vim
Sourced 1 time
Total time:   0.000157
 Self time:   0.000157

count  total (s)   self (s)
                            
    3              0.000020 for i in range(2)
                            
    2              0.000003   if i == 1
    1              0.000003     let a = round(0.5)
    1              0.000015     echo a
    1              0.000001   else
    1              0.000015     if x == 1 | let b = len([]) | endif
                                echo b
    1              0.000002   endif
    2              0.000009 endfor
                            

