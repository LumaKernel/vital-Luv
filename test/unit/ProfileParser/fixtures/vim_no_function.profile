SCRIPT  /home/luma/vim-make-plugin/vital-Luv/dev/ProfileParser/test_no_function.vim
Sourced 1 time
Total time:   0.000388
 Self time:   0.000388

count  total (s)   self (s)
                            
    3              0.000025 for i in range(2)
                            
    2              0.000021   if i == 1
    1              0.000004     let a = round(0.5)
    1              0.000027     echo a
    1              0.000002   else
    1              0.000047     if x == 1 | let b = len([]) | endif
                                echo b
    1              0.000001   endif
    2              0.000008 endfor
                            

