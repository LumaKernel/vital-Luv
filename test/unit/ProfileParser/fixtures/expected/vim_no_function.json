[{"path":"/home/luma/vim-make-plugin/vital-Luv/dev/ProfileParser/test_no_function.vim","total_time":1.57e-4,"self_time":1.57e-4,"type":"script","lines":[{"content":""},{"self_time":2.0e-5,"count":3,"content":"for i in range(2)"},{"content":""},{"self_time":3.0e-6,"count":2,"content":"  if i == 1"},{"self_time":3.0e-6,"count":1,"content":"    let a = round(0.5)"},{"self_time":1.5e-5,"count":1,"content":"    echo a"},{"self_time":1.0e-6,"count":1,"content":"  else"},{"self_time":1.5e-5,"count":1,"content":"    if x == 1 | let b = len([]) | endif"},{"content":"    echo b"},{"self_time":2.0e-6,"count":1,"content":"  endif"},{"self_time":9.0e-6,"count":2,"content":"endfor"},{"content":""}],"count":1}]
